schema/schema_merge.go

```diff
@@ -19,18 +19,15 @@ import (
 
 type SchemaMerger struct {
 	coreSchema       *schema.BodySchema
-	schemaReader     SchemaReader
 	terraformVersion *version.Version
-	moduleReader     ModuleReader
+	stateReader      StateReader
 }
 
-type ModuleReader interface {
-	ModuleCalls(modPath string) (tfmod.ModuleCalls, error)
+type StateReader interface {
```

Comment:
I'm sure I just forgot, but maybe a comment might help here to what a "state" is to terraform-schema and what the state reader handles?